Common/Brokerages/TDameritradeBrokerageModel.cs

```diff
@@ -0,0 +1,120 @@
+        /// <returns>True if the brokerage could process the order, false otherwise</returns>
+        public override bool CanSubmitOrder(Security security, Order order, out BrokerageMessageEvent message)
+        {
+            if (!IsValidOrderSize(security, order.Quantity, out message))
+            {
+                return false;
+            }
+
+            message = null;
+            if (!new[] { SecurityType.Equity, SecurityType.Option }.Contains(security.Type))
+            {
+                message = new BrokerageMessageEvent(BrokerageMessageType.Warning, "NotSupported",
+                    StringExtensions.Invariant($"The {nameof(TDAmeritradeBrokerageModel)} does not support {security.Type} security type.")
+                );
+
+                return false;
+            }
+
+            if (!new[] { OrderType.Market, OrderType.Limit, OrderType.StopMarket, OrderType.StopLimit }.Contains(order.Type))
+            {
+                message = new BrokerageMessageEvent(BrokerageMessageType.Warning, "NotSupported",
+                    StringExtensions.Invariant($"{order.Type} order is not supported by TDAmeritrade. Currently, only Market Order is supported.")
+                );
+
+                return false;
+            }
+
+            return base.CanSubmitOrder(security, order, out message);
+        }
+
+        /// <summary>
+        /// TDAmeritrade does not support update of orders
+        /// </summary>
+        /// <param name="security">Security</param>
+        /// <param name="order">Order that should be updated</param>
+        /// <param name="request">Update request</param>
+        /// <param name="message">Outgoing message</param>
+        /// <returns>Always false as TDAmeritrade does not support update of orders</returns>
+        public override bool CanUpdateOrder(Security security, Order order, UpdateOrderRequest request, out BrokerageMessageEvent message)
+        {
+            message = new BrokerageMessageEvent(BrokerageMessageType.Warning, 0, "Brokerage does not support update. You must cancel and re-create instead."); ;
+            return false;
+        }
+
+        /// <summary>
+        /// Provides TDAmeritrade fee model
+        /// </summary>
+        /// <param name="security">Security</param>
+        /// <returns>TDAmeritrade fee model</returns>
+        public override IFeeModel GetFeeModel(Security security)
+        {
+            return new TDAmeritradeFeeModel();
+        }
+
+        /// <summary>
+        /// Get default markets and specify TDAmeritrade as crypto market
+        /// </summary>
+        /// <returns>default markets</returns>
+        private static IReadOnlyDictionary<SecurityType, string> GetDefaultMarkets()
+        {
+            var map = DefaultMarketMap.ToDictionary();
+            map[SecurityType.Equity] = Market.USA;
+            return map.ToReadOnlyDictionary();
```

Comment:
>  map[SecurityType.Equity] = Market.USA;

Think this is already the default? Maybe can avoid doing something special and rely on the default